contract/cmdb.sol

```diff
@@ -27,8 +27,10 @@ contract CMDb {
     // Add a rating on a movie
     // Parameters: the movie title, the user's rating
     function addRating(string memory _title, uint _rating) public {
-        if(!movies[_title].exists) return; // If the movie doesn't exist, do nothing
-        if(_rating < 0 || _rating > 10) return; // If the rating isn't on a scale of 1 to 10, do nothing
+        // if(!movies[_title].exists) return; // If the movie doesn't exist, do nothing
+        require(movies[_title].exists, "This movie does not exist");
+        // if(_rating < 0 || _rating > 10) return; // If the rating isn't on a scale of 1 to 10, do nothing
+        require(_rating > 0 || _rating < 10, "Please enter a valid rating");
```

Comment:
Should be
`require(_rating >= 0 && _rating <= 10, "Please enter a valid rating");`